### PR TITLE
multiple changes

### PR DESCRIPTION
**Pull request by aakash-v-virani**
Signed-off-by: aakash-v-virani <aakash.v.virani@intel.com>

### CODE REVIEW

File: asr_deepspeech/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ This service is concerned with converting speech input to text output. It can re
 
 ## Pre-requisites
 - 0MQ
-- OpenVINO&trade; 2021.3
+- OpenVINO&trade; 2020.4
```
Comment (vinayg-intel) [inline asr_deepspeech/README.md]: openvino/ubuntu18_runtime:2021.3 is used

Comment (vinayg-intel) [reply asr_deepspeech/README.md]: https://github.com/intel-innersource/applications.ai.conversational-ai.chatbot/blob/master/dockerfiles/deepspeech8_asr.dockerfile

File: asr_kaldi/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 ## Pre-requisites
 - 0MQ
-- OpenVINO&trade; 2021.3
+- OpenVINO&trade; 2020.4
```
Comment (vinayg-intel) [inline asr_kaldi/README.md]: 2021.3
https://github.com/intel-innersource/applications.ai.conversational-ai.chatbot/blob/master/dockerfiles/kaldi_asr.dockerfile

File: authz/README.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ python main.py
 
 ## Pre-requisites
 - 0MQ
-- OpenVINO&trade; 2021.3
+- OpenVINO&trade; 2020.4
```
Comment (vinayg-intel) [inline authz/README.md]: not require to mention about openvino

**Review by vinayg-intel — CHANGES_REQUESTED** [review]

File: asr_deepspeech/src/speech_library/examples/speech2text.py [hunk 4]
```diff
@@ -52,22 +52,28 @@ def main():
...
     print(time.time() - t3)
 
-    if r1 and output_file:
-        with open(output_file, "w") as f:
-            f.write(r)
-    else:
-        print("transcribed text: {}".format(r))
+
+    if r1:
+        print("transcribed text: Please uncomment the 68th line")
```
Comment (vinayg-intel) [inline asr_deepspeech/src/speech_library/examples/speech2text.py]: line number incorrect

**Review by vinayg-intel — CHANGES_REQUESTED** [review]

File: asr_deepspeech/src/speech_library/examples/speech2text.py [hunk 5]
```diff
@@ -52,22 +52,28 @@ def main():
...
+            if f.closed == False:
+                f.close()
+
 
     t3 = time.time()
     sm.push_data(data, finish_processing=True)
     r, r1 = sm.get_result()
     print(time.time() - t3)
 
-    if r1 and output_file:
```
Comment (vinayg-intel) [inline asr_deepspeech/src/speech_library/examples/speech2text.py:66]: need to see if these lines need to be commented

**Review by vinayg-intel — CHANGES_REQUESTED** [review]

File: asr_deepspeech/README.md (hunk 1 above)
Comment (aakash-v-virani) [reply asr_deepspeech/README.md]: fixed in cabf1272c5240e39836103f5147554ca8ea86acb

File: asr_kaldi/README.md (hunk 2 above)
Comment (aakash-v-virani) [reply asr_kaldi/README.md]: fixed in cabf1272c5240e39836103f5147554ca8ea86acb

File: asr_deepspeech/src/speech_library/examples/speech2text.py (hunk 4 above)
Comment (aakash-v-virani) [reply asr_deepspeech/src/speech_library/examples/speech2text.py]: fixed in cabf1272c5240e39836103f5147554ca8ea86acb

File: asr_deepspeech/src/speech_library/examples/speech2text.py (hunk 5 above)
Comment (aakash-v-virani) [reply asr_deepspeech/src/speech_library/examples/speech2text.py:66]: fixed in cabf1272c5240e39836103f5147554ca8ea86acb

File: authz/README.md (hunk 3 above)
Comment (aakash-v-virani) [reply authz/README.md]: fixed in bb9995e1a37b21ca5fbe7d13de2bfa0f470a692a

**Review by vinayg-intel — APPROVED** [review]

**Pull request merged**